cmd/registry/cmd/annotate_test.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestAnnotate(t *testing.T) {
+	var err error
+
+	projectId := "annotate-test"
+	projectName := "projects/" + projectId
+	apiId := "sample"
+	apiName := projectName + "/apis/" + apiId
+	versionId := "1.0.0"
+	versionName := apiName + "/versions/" + versionId
+	specId := "openapi.json"
+	specName := versionName + "/specs/" + specId
```

Comment:
Consider using a const block for the IDs and other "irrelevant" test data where the contents aren't a key piece of what's being tested. 

Style for initialisms should be to [maintain uppercase](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). Does `golint` warn you about these? It usually does for me.
```
const (
    projectID = "annotate-test"
    apiID = "sample"
    versionID = "1.0.0"
    specID = "openapi.json"
    openAPIv3 = "application/x.openapi;version=3.0.0"
    specContents = `{"openapi": "3.0.0", "info": {"title": "test", "version": "v1"}, "paths": {}}`
)
```

Instead of discarding the created project, api, version, and spec you can keep the result and use GetName() in the tests. You might not need custom resource IDs at all actually. At a glance it looks like the tests will work even with generated IDs.

Reply:
Done (partly) - I've updated to the const style and initialisms but kept the custom ids (will ping you about that)